files/project_team/IFCTunnelDep2.2.ifc

```diff
@@ -0,0 +1,630 @@
+#20= IFCAXIS2PLACEMENT3D(#19,$,$);
+#21= IFCLOCALPLACEMENT(#18,#20);
+#22= IFCTUNNELPART('3qT_1GVJjBnQwCN_qI7a_N',#10,$,$,$,#21,$,$,$,.LONGITUDINAL.,.TUNNELSECTION.);
+#23= IFCRELAGGREGATES('29qONl9i0jZ2yHi6Jc3U5Q',#10,$,$,#14,(#22,#27));
+#24= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#25= IFCAXIS2PLACEMENT3D(#24,$,$);
+#26= IFCLOCALPLACEMENT(#18,#25);
+#27= IFCTUNNELPART('0viQbYyAL0xxca2xjwsVC1',#10,$,$,$,#26,$,$,$,.LONGITUDINAL.,.TUNNELSECTION.);
+#28= IFCALIGNMENT('02N$NVmQ94BvRFE4JZGxNH',$,'E, oID:2','315',$,#13,#550,$);
+#29= IFCPROPERTYSINGLEVALUE('Status',$,IFCLABEL('NEW'),$);
+#30= IFCPROPERTYSET('0qp0JANF53Fh7PG6Fdw0y$',$,'Pset_CivilElementCommon',$,(#29));
+#31= IFCRELDEFINESBYPROPERTIES('3eQRkdqjj2cBDYhAZqQH8q',#10,$,$,(#28),#30);
+#32= IFCRELNESTS('2SBA2P$JD2sR2Xrw4PHolF',$,'name',$,#28,(#33,#144,#359));
+#33= IFCALIGNMENTHORIZONTAL('1INPHBCd19EwZAMY0j_Qun',$,$,'VMU',$,#138,#141);
+#34= IFCRELNESTS('1MB$C_BUX0bPGhzYQd6LH3',$,'name',$,#33,(#35,#58,#74,#90,#106,#122));
+#35= IFCALIGNMENTSEGMENT('3DykKH4A18gQFgDBppHg44',$,'E',$,$,#36,#39,#56);
+#36= IFCLOCALPLACEMENT($,#37);
+#37= IFCAXIS2PLACEMENT3D(#38,$,$);
+#38= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#39= IFCPRODUCTDEFINITIONSHAPE($,$,(#40));
+#40= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#48));
+#41= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,0.0000000254,#42,#561);
+#42= IFCAXIS2PLACEMENT3D(#43,#44,#46);
+#43= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#44= IFCDIRECTION((0.0,0.0,1.0));
+#45= IFCDIRECTION((1.0,0.0,0.0));
+#46= IFCDIRECTION((1.0,0.0,0.0));
+#47= IFCDIRECTION((0.0,1.0));
+#48= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#49,IFCNONNEGATIVELENGTHMEASURE(0.0),IFCNONNEGATIVELENGTHMEASURE(1508.3410985277),#52);
+#49= IFCAXIS2PLACEMENT2D(#50,#51);
+#50= IFCCARTESIANPOINT((162572.948873,6721939.071596));
+#51= IFCDIRECTION((-0.99484944,-0.10136364));
+#52= IFCLINE(#53,#54);
+#53= IFCCARTESIANPOINT((0.0,0.0));
+#54= IFCVECTOR(#55,1.0);
+#55= IFCDIRECTION((-0.99484944,-0.10136364));
+#56= IFCALIGNMENTHORIZONTALSEGMENT(', code=POB',', code=TS',#57,-3.04005462523801,0.0,0.0,1508.341099,$,.LINE.);
+#57= IFCCARTESIANPOINT((162572.948873,6721939.071596));
+#58= IFCALIGNMENTSEGMENT('3f3U42JTHDjAX1u4qJChx$',$,'E',$,$,#59,#62,#72);
+#59= IFCLOCALPLACEMENT($,#60);
+#60= IFCAXIS2PLACEMENT3D(#61,$,$);
+#61= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#62= IFCPRODUCTDEFINITIONSHAPE($,$,(#63));
+#63= IFCSHAPEREPRESENTATION(#41,'Axis','Segment',(#64));
+#64= IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE.,#65,IFCPARAMETERVALUE(0.0),IFCNONNEGATIVELENGTHMEASURE(92.0),#68);
+#65= IFCAXIS2PLACEMENT2D(#66,#67);
+#66= IFCCARTESIANPOINT((161072.376573,6721786.180648));
+#67= IFCDIRECTION((-0.99484944,-0.10136365));
+#68= IFCCLOTHOID(#69,418.090899207337);
+#69= IFCAXIS2PLACEMENT2D(#70,#71);
+#70= IFCCARTESIANPOINT((0.0,0.0));
+#71= IFCDIRECTION((-0.99484944,-0.10136365));
+#72= IFCALIGNMENTHORIZONTALSEGMENT(', code=TS',', code=SC',#73,-3.04005461353714,0.0,1900.0,92.0,$,.CLOTHOID.);
```

Comment:
Interesting tags.

Reply:
> Interesting tags.

Comes from the LandXML and the LandXML=>IFC converter from Peter Bonsma.